std/traits.d

```diff
@@ -4125,6 +4125,13 @@ unittest
     static assert(is(ImplicitConversionTargets!(string)[0] == const(char)[]));
 }
 
+unittest // bugzilla 15940
+{
+    class Foo{}
+    struct Bar{Foo foo; alias foo this;}
+    static assert(ImplicitConversionTargets!(Bar).length == 0);
```

Comment:
But `Bar` does convert implicitly to `Foo`. So `ImplicitConversionTargets!Bar` should list `Foo`, shouldn't it? Although it looks like `ImplicitConversionTargets` doesn't recognize `alias this` at all.